app/styles/main.css

```diff
@@ -1369,3 +1369,21 @@ label { position: relative; }
 .channelListingTable tr:hover .channelPriorityOrdering {
   display: block;
 }
+
+.rolesChannelsMatrixDiv .isAllowed { color: #009933; margin: 0px 2px; font-size: 13px; }
+.rolesChannelsMatrixDiv .isNotAllowed { color: #FF0000; margin: 0px 2px; font-size: 13px; }
+
+.limitChannelTdWidth {width:120px;}
+
+.roleAssignedToClientSpan { color: #333333; }
+.roleNotAssignedToClientSpan { color: #999; text-decoration: line-through; }
+
+.cursorPointer {cursor: pointer}
```

Comment:
Just a style comment, please put spaces after your `{` as done above so it stays consistent.